.github/workflows/windows.yml

```diff
@@ -42,8 +40,6 @@ jobs:
       run: choco install nasm
     - name: Add Nasm to PATH
       run: echo "::add-path::C:\Program Files\Nasm"
-#    - name: test nasm
-#      run: nasm.exe -v
```

Comment:
why?

Reply:
It was commented, but I could just uncomment if you prefer.

Reply:
ohhh, right..duh...You can go on